recipes/libtheora/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "1.1.1" %}
+
+package:
+  name: libtheora
+  version: {{ version }}
+
+source:
+  fn: libtheora-{{ version }}.tar.bz2
+  url: http://downloads.xiph.org/releases/theora/libtheora-{{ version }}.tar.bz2
+  md5: 292ab65cedd5021d6b7ddd117e07cd8e
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+  build:
+    - pkg-config
+    - toolchain
+    - libogg 1.3.*
+    - libvorbis 1.3.*
+    - libpng >=1.6.23,<1.7
+    - zlib 1.2.*
+
+  run:
+    - libogg 1.3.*
+    - libvorbis 1.3.*
+    - libpng >=1.6.23,<1.7
+    - zlib 1.2.*
+
+test:
+  commands:
+    # Verify libraries.
+    {% set libtheora_libs = [
+        "theora",
+        "theoradec",
+        "theoraenc"
+    ] %}
+    {% for each_libtheora_lib in libtheora_libs %}
+    - test -f ${PREFIX}/lib/lib{{ each_libtheora_lib }}.a                           # [unix]
+    - test -f ${PREFIX}/lib/lib{{ each_libtheora_lib }}.dylib                       # [osx]
+    - test -f ${PREFIX}/lib/lib{{ each_libtheora_lib }}.so                          # [linux]
+    {% endfor %}
+
+
+about:
+  home: https://theora.org
+  license: BSD 3-Clause
+  # BSD like
+  license_family: BSD
+  # Additional licencing information in LICENSE as well
+  license_file: COPYING
```

Comment:
Yeah, I was searching for it, but there is no mechanism. Ideally the upstream package should consolidate the License information.